.github/workflows/review.yml

```diff
@@ -38,6 +38,9 @@ jobs:
     runs-on: ubuntu-latest
     needs: build
     name: Review
+    # allow only one deployment at a time
+    concurrency: ${{ github.workflow }}-${{ github.head_ref }}
```

Comment:
what happens if I merge 2 pull requests? Will them go in queue or will one of them get dismissed?

Reply:
Review workflow
- pull requests are run separately and parallel like now. 
- 2 (or more) commits to same pull request at the same time:
  - build -steps are run parallel as now
  - deploy -steps to Kubernetes are run on queue

Staging workflow
- merge 2 (or more) pull request at the same time to master triggers 2 actions where:
  - build -steps are run parallel as now
  - deploy -steps to Kubernetes are run on queue

Production workflow
- all release actions are run on queue (this is more theoretical case)